cfgmgr/teammgr.cpp

```diff
@@ -112,18 +115,48 @@ void TeamMgr::doTask(Consumer &consumer)
     }
 }
 
-
 void TeamMgr::cleanTeamProcesses()
 {
     SWSS_LOG_ENTER();
     SWSS_LOG_NOTICE("Cleaning up LAGs during shutdown...");
-    for (const auto& it: m_lagList)
+
+    struct timespec req = {.tv_sec = 0, .tv_nsec = 100000}; // 100 usec
+
+    for (const auto& alias: m_lagList)
     {
-        //This will call team -k kill -t <teamdevicename> which internally send SIGTERM 
-        removeLag(it);
+        pid_t pid;
+
+        if (!getLagPid(pid, alias))
+        {
+            continue;
+        }
+
+        SWSS_LOG_INFO("Send SIGTERM to port channel %s pid %d", alias.c_str(), pid);
+
+        if (kill(pid, SIGTERM) < 0)
```

Comment:
@judyjoseph FYI. These warnings:
```bash
Aug 17 17:58:40.464776 sonic WARNING kernel: [ 3617.296910] PortChannel2 (unregistering): Failed to send options change via netlink (err -105)
Aug 17 17:58:40.464796 sonic WARNING kernel: [ 3617.297646] PortChannel62 (unregistering): Failed to send options change via netlink (err -105)
Aug 17 17:58:40.464803 sonic WARNING kernel: [ 3617.298464] PortChannel53 (unregistering): Failed to send options change via netlink (err -105)
```
can be fixed in two ways:
1. Adding sleep after `SIGTERM` is sent
2. Improving `teamd/kernel` behaviour to avoid race

Reply:
@judyjoseph opinion?

Reply:
Thanks @nazariig for trying out this approach to use the signal API calls. The err -105 points to "no buffer space available" in the stack and which will result in netlink messages not getting delivered correctly. Adding a sleep between consecutive SIGTERM could solve the issue, but may be might introduce some new timing issues ?

Meanwhile I tried an alternative approach to NOT doing an EXEC call per portchannel  "teamd -k -t PortChannel000". Rather append that command to a string -- and issue the commands together as a list in a single bash shell. Snip of the code is below ( this is sample code, needs to be made PR ready). I see that the Portchannels gets cleaned up in same second at least in the testbed I used with 4 Portchannels.   

I suspect then spawning separate bash shell was causing cleanup delay in master branch ? I need to verify with more number of portchannels though.

```
diff --git a/cfgmgr/teammgr.cpp b/cfgmgr/teammgr.cpp
index a0f7c41..85becd9 100644
--- a/cfgmgr/teammgr.cpp
+++ b/cfgmgr/teammgr.cpp
@@ -116,13 +116,19 @@ void TeamMgr::doTask(Consumer &consumer)
 void TeamMgr::cleanTeamProcesses()
 {
     SWSS_LOG_ENTER();
+    std::string cmd;
+    string res;
+
     SWSS_LOG_NOTICE("Cleaning up LAGs during shutdown...");
+
     for (const auto& it: m_lagList)
     {
         //This will call team -k kill -t <teamdevicename> which internally send SIGTERM 
-        removeLag(it);
+        removeLag(it, cmd);
     }
 
+    EXEC_WITH_ERROR_THROW(cmd, res);
+
     return;
 }
 
@@ -214,7 +220,8 @@ void TeamMgr::doLagTask(Consumer &consumer)
         {
             if (m_lagList.find(alias) != m_lagList.end())
             {
-                removeLag(alias);
+               string cmd;
+                removeLag(alias, cmd);
                 m_lagList.erase(alias);
             }
         }
@@ -515,15 +522,15 @@ task_process_status TeamMgr::addLag(const string &alias, int min_links, bool fal
     return task_success;
 }
 
-bool TeamMgr::removeLag(const string &alias)
+bool TeamMgr::removeLag(const string &alias, string &cmd_all)
 {
     SWSS_LOG_ENTER();
 
     stringstream cmd;
     string res;
 
-    cmd << TEAMD_CMD << " -k -t " << shellquote(alias);
-    EXEC_WITH_ERROR_THROW(cmd.str(), res);
+    cmd << TEAMD_CMD << " -k -t " << shellquote(alias) << "; ";
+    cmd_all.append(cmd.str());
 
     SWSS_LOG_NOTICE("Stop port channel %s", alias.c_str());
 
diff --git a/cfgmgr/teammgr.h b/cfgmgr/teammgr.h
index 4d2f20b..304d67c 100644
--- a/cfgmgr/teammgr.h
+++ b/cfgmgr/teammgr.h
@@ -42,7 +42,7 @@ private:
     void doPortUpdateTask(Consumer &consumer);
 
     task_process_status addLag(const std::string &alias, int min_links, bool fall_back);
-    bool removeLag(const std::string &alias);
+    bool removeLag(const std::string &alias, std::string &cmd_all);
     task_process_status addLagMember(const std::string &lag, const std::string &member);
     bool removeLagMember(const std::string &lag, const std::string &member);
```

Reply:
@judyjoseph the suggest patch doesn't fix the original bug:
```bash
Aug 25 14:52:47.152136 sonic NOTICE teamd#teammgrd: :- cleanTeamProcesses: Cleaning up LAGs during shutdown...
Aug 25 14:52:47.152136 sonic NOTICE teamd#teammgrd: :- cleanTeamProcesses: Patch by judyjoseph - aggregate remove approach

Aug 25 14:52:52.162687 sonic INFO teamd#supervisord 2021-08-25 14:52:52,161 INFO reaped unknown pid 58 (exit status 0)
Aug 25 14:52:52.224096 sonic NOTICE swss#portsyncd: :- onMsg: nlmsg type:16 key:PortChannel14 admin:0 oper:0 addr:1c:34:da:1d:42:00 ifindex:415 master:0 type:team
Aug 25 14:52:52.227377 sonic NOTICE swss#portsyncd: :- onMsg: nlmsg type:17 key:PortChannel14 admin:0 oper:0 addr:1c:34:da:1d:42:00 ifindex:415 master:0 type:team
Aug 25 14:52:53.165429 sonic INFO teamd#supervisord 2021-08-25 14:52:53,163 INFO reaped unknown pid 66 (exit status 0)
Aug 25 14:52:53.237513 sonic NOTICE swss#portsyncd: :- onMsg: nlmsg type:16 key:PortChannel15 admin:0 oper:0 addr:1c:34:da:1d:42:00 ifindex:416 master:0 type:team
Aug 25 14:52:53.240298 sonic NOTICE swss#portsyncd: :- onMsg: nlmsg type:17 key:PortChannel15 admin:0 oper:0 addr:1c:34:da:1d:42:00 ifindex:416 master:0 type:team

Aug 25 14:53:42.336812 sonic INFO dockerd[601]: time="2021-08-25T14:53:42.334519626Z" level=info msg="Container 484888a93a601dc6e9b7e6f26e2d3d8b8da750691db88333502f48138b10b032 failed to exit within 60 seconds of signal 15 - using the force"
```

The results you have got is just a logs without an actual call of LAG cleanup.

```bash
root@sonic:/home/admin# time service teamd stop

real    1m1.615s
user    0m0.271s
sys     0m0.151s

root@sonic:/home/admin# tail -F /var/log/syslog

Aug 25 14:52:08.265229 sonic INFO systemd-udevd[55004]: message repeated 38 times: [ ethtool: autonegotiation is unset or enabled, the speed and duplex are not writable.]
Aug 25 14:52:41.017998 sonic INFO systemd[1]: Started Host config enforcer daemon.
Aug 25 14:52:41.020086 sonic INFO systemd[1]: Started AAA Statistics Collection daemon.
Aug 25 14:52:41.081278 sonic WARNING systemd[1]: Configuration file /lib/systemd/system/copp-config.service is marked executable. Please remove executable permission bits. Proceeding anyway.
Aug 25 14:52:41.096996 sonic INFO /aaastatsd: ConfigDB connect success
Aug 25 14:52:41.097303 sonic INFO /aaastatsd: CountersDB connect success
Aug 25 14:52:41.097564 sonic INFO /aaastatsd: RADIUS Stats File Monitor started
Aug 25 14:52:41.188920 sonic INFO /hostcfgd: ConfigDB connect success
Aug 25 14:52:41.189232 sonic INFO /hostcfgd: KdumpCfg load ...
Aug 25 14:52:41.189474 sonic INFO /hostcfgd: Kdump global configuration update
Aug 25 14:52:41.835748 sonic INFO systemd[1]: Stopping TEAMD container...
Aug 25 14:52:41.841255 sonic NOTICE admin: Stopping teamd service...
Aug 25 14:52:41.861883 sonic NOTICE kdump-config: unloaded kdump kernel
Aug 25 14:52:42.029739 sonic INFO hostcfgd[56478]: kdump is already disabled
Aug 25 14:52:42.029944 sonic INFO hostcfgd[56478]: Kdump is already disabled
Aug 25 14:52:42.048237 sonic NOTICE admin: Warm boot flag: teamd false.
Aug 25 14:52:42.055271 sonic NOTICE admin: Fast boot flag: teamd false.
Aug 25 14:52:42.289487 sonic DEBUG /container: container_stop: BEGIN
Aug 25 14:52:42.289811 sonic DEBUG /container: read_data: config:True feature:teamd fields:[('set_owner', 'local'), ('no_fallback_to_local', False)] val:['local', False]
Aug 25 14:52:42.290218 sonic DEBUG /container: read_data: config:False feature:teamd fields:[('current_owner', 'none'), ('remote_state', 'none'), ('container_id', '')] val:['none', 'none', '']
Aug 25 14:52:42.290718 sonic DEBUG /container: container_stop: teamd: set_owner:local current_owner:none remote_state:none docker_id:teamd
Aug 25 14:52:42.429510 sonic INFO hostcfgd[56583]: Kdump configuration has been updated in the startup configuration
Aug 25 14:52:42.835013 sonic INFO hostcfgd[56643]: Kdump configuration has been updated in the startup configuration
Aug 25 14:52:42.843447 sonic INFO /hostcfgd: Waiting for systemctl to finish initialization
Aug 25 14:52:42.867554 sonic INFO /hostcfgd: systemctl has finished initialization -- proceeding ...
Aug 25 14:52:42.895663 sonic INFO systemd[1]: Reloading.
Aug 25 14:52:43.181124 sonic NOTICE systemd[1]: /lib/systemd/system/kdump-tools.service:8: Standard output type syslog+console is obsolete, automatically updating to journal+console. Please update your unit file, and consider removing the setting altogether.
Aug 25 14:52:43.190685 sonic WARNING systemd[1]: Configuration file /lib/systemd/system/copp-config.service is marked executable. Please remove executable permission bits. Proceeding anyway.
Aug 25 14:52:43.330174 sonic INFO teamd#supervisord 2021-08-25 14:52:43,329 WARN received SIGTERM indicating exit request
Aug 25 14:52:43.330970 sonic INFO teamd#supervisord 2021-08-25 14:52:43,330 INFO waiting for supervisor-proc-exit-listener, rsyslogd, teammgrd, teamsyncd, tlm_teamd to die
Aug 25 14:52:43.334214 sonic INFO /hostcfgd: Feature bgp is enabled and started
Aug 25 14:52:43.356157 sonic INFO systemd[1]: Reloading.
Aug 25 14:52:43.636071 sonic NOTICE systemd[1]: /lib/systemd/system/kdump-tools.service:8: Standard output type syslog+console is obsolete, automatically updating to journal+console. Please update your unit file, and consider removing the setting altogether.
Aug 25 14:52:43.645624 sonic WARNING systemd[1]: Configuration file /lib/systemd/system/copp-config.service is marked executable. Please remove executable permission bits. Proceeding anyway.
Aug 25 14:52:43.796201 sonic INFO /hostcfgd: Feature database is enabled and started
Aug 25 14:52:43.816330 sonic INFO systemd[1]: Reloading.
Aug 25 14:52:44.086788 sonic NOTICE systemd[1]: /lib/systemd/system/kdump-tools.service:8: Standard output type syslog+console is obsolete, automatically updating to journal+console. Please update your unit file, and consider removing the setting altogether.
Aug 25 14:52:44.095219 sonic WARNING systemd[1]: Configuration file /lib/systemd/system/copp-config.service is marked executable. Please remove executable permission bits. Proceeding anyway.
Aug 25 14:52:44.228658 sonic INFO /hostcfgd: Feature dhcp_relay is enabled and started
Aug 25 14:52:44.247043 sonic INFO systemd[1]: Reloading.
Aug 25 14:52:44.343249 sonic NOTICE teamd#tlm_teamd: :- main: Exiting
Aug 25 14:52:44.343249 sonic NOTICE teamd#tlm_teamd: :- ~TeamdCtlMgr: Exiting. Disconnecting from teamd. LAG 'PortChannel9'
Aug 25 14:52:44.343331 sonic NOTICE teamd#tlm_teamd: :- ~TeamdCtlMgr: Exiting. Disconnecting from teamd. LAG 'PortChannel8'
Aug 25 14:52:44.343475 sonic NOTICE teamd#tlm_teamd: :- ~TeamdCtlMgr: Exiting. Disconnecting from teamd. LAG 'PortChannel63'
Aug 25 14:52:44.343502 sonic NOTICE teamd#tlm_teamd: :- ~TeamdCtlMgr: Exiting. Disconnecting from teamd. LAG 'PortChannel62'
Aug 25 14:52:44.343567 sonic NOTICE teamd#tlm_teamd: :- ~TeamdCtlMgr: Exiting. Disconnecting from teamd. LAG 'PortChannel57'
Aug 25 14:52:44.343699 sonic NOTICE teamd#tlm_teamd: :- ~TeamdCtlMgr: Exiting. Disconnecting from teamd. LAG 'PortChannel56'
Aug 25 14:52:44.343699 sonic NOTICE teamd#tlm_teamd: :- ~TeamdCtlMgr: Exiting. Disconnecting from teamd. LAG 'PortChannel55'
Aug 25 14:52:44.343699 sonic NOTICE teamd#tlm_teamd: :- ~TeamdCtlMgr: Exiting. Disconnecting from teamd. LAG 'PortChannel53'
Aug 25 14:52:44.343810 sonic NOTICE teamd#tlm_teamd: :- ~TeamdCtlMgr: Exiting. Disconnecting from teamd. LAG 'PortChannel52'
Aug 25 14:52:44.343933 sonic NOTICE teamd#tlm_teamd: :- ~TeamdCtlMgr: Exiting. Disconnecting from teamd. LAG 'PortChannel50'
Aug 25 14:52:44.344012 sonic NOTICE teamd#tlm_teamd: :- ~TeamdCtlMgr: Exiting. Disconnecting from teamd. LAG 'PortChannel6'
Aug 25 14:52:44.344012 sonic NOTICE teamd#tlm_teamd: :- ~TeamdCtlMgr: Exiting. Disconnecting from teamd. LAG 'PortChannel47'
Aug 25 14:52:44.344352 sonic NOTICE teamd#tlm_teamd: :- ~TeamdCtlMgr: Exiting. Disconnecting from teamd. LAG 'PortChannel46'
Aug 25 14:52:44.344352 sonic NOTICE teamd#tlm_teamd: :- ~TeamdCtlMgr: Exiting. Disconnecting from teamd. LAG 'PortChannel43'
Aug 25 14:52:44.344398 sonic NOTICE teamd#tlm_teamd: :- ~TeamdCtlMgr: Exiting. Disconnecting from teamd. LAG 'PortChannel5'
Aug 25 14:52:44.344398 sonic NOTICE teamd#tlm_teamd: :- ~TeamdCtlMgr: Exiting. Disconnecting from teamd. LAG 'PortChannel58'
Aug 25 14:52:44.344474 sonic NOTICE teamd#tlm_teamd: :- ~TeamdCtlMgr: Exiting. Disconnecting from teamd. LAG 'PortChannel49'
Aug 25 14:52:44.344821 sonic NOTICE teamd#tlm_teamd: :- ~TeamdCtlMgr: Exiting. Disconnecting from teamd. LAG 'PortChannel48'
Aug 25 14:52:44.344821 sonic NOTICE teamd#tlm_teamd: :- ~TeamdCtlMgr: Exiting. Disconnecting from teamd. LAG 'PortChannel22'
Aug 25 14:52:44.344855 sonic NOTICE teamd#tlm_teamd: :- ~TeamdCtlMgr: Exiting. Disconnecting from teamd. LAG 'PortChannel32'
Aug 25 14:52:44.344951 sonic NOTICE teamd#tlm_teamd: :- ~TeamdCtlMgr: Exiting. Disconnecting from teamd. LAG 'PortChannel45'
Aug 25 14:52:44.344951 sonic NOTICE teamd#tlm_teamd: :- ~TeamdCtlMgr: Exiting. Disconnecting from teamd. LAG 'PortChannel2'
Aug 25 14:52:44.344951 sonic NOTICE teamd#tlm_teamd: :- ~TeamdCtlMgr: Exiting. Disconnecting from teamd. LAG 'PortChannel18'
Aug 25 14:52:44.345252 sonic NOTICE teamd#tlm_teamd: :- ~TeamdCtlMgr: Exiting. Disconnecting from teamd. LAG 'PortChannel44'
Aug 25 14:52:44.345354 sonic NOTICE teamd#tlm_teamd: :- ~TeamdCtlMgr: Exiting. Disconnecting from teamd. LAG 'PortChannel41'
Aug 25 14:52:44.345418 sonic NOTICE teamd#tlm_teamd: :- ~TeamdCtlMgr: Exiting. Disconnecting from teamd. LAG 'PortChannel64'
Aug 25 14:52:44.345418 sonic NOTICE teamd#tlm_teamd: :- ~TeamdCtlMgr: Exiting. Disconnecting from teamd. LAG 'PortChannel20'
Aug 25 14:52:44.345726 sonic NOTICE teamd#tlm_teamd: :- ~TeamdCtlMgr: Exiting. Disconnecting from teamd. LAG 'PortChannel19'
Aug 25 14:52:44.345765 sonic NOTICE teamd#tlm_teamd: :- ~TeamdCtlMgr: Exiting. Disconnecting from teamd. LAG 'PortChannel13'
Aug 25 14:52:44.345797 sonic NOTICE teamd#tlm_teamd: :- ~TeamdCtlMgr: Exiting. Disconnecting from teamd. LAG 'PortChannel17'
Aug 25 14:52:44.345797 sonic NOTICE teamd#tlm_teamd: :- ~TeamdCtlMgr: Exiting. Disconnecting from teamd. LAG 'PortChannel10'
Aug 25 14:52:44.345821 sonic NOTICE teamd#tlm_teamd: :- ~TeamdCtlMgr: Exiting. Disconnecting from teamd. LAG 'PortChannel59'
Aug 25 14:52:44.345950 sonic NOTICE teamd#tlm_teamd: :- ~TeamdCtlMgr: Exiting. Disconnecting from teamd. LAG 'PortChannel42'
Aug 25 14:52:44.345980 sonic NOTICE teamd#tlm_teamd: :- ~TeamdCtlMgr: Exiting. Disconnecting from teamd. LAG 'PortChannel37'
Aug 25 14:52:44.346054 sonic NOTICE teamd#tlm_teamd: :- ~TeamdCtlMgr: Exiting. Disconnecting from teamd. LAG 'PortChannel54'
Aug 25 14:52:44.346128 sonic NOTICE teamd#tlm_teamd: :- ~TeamdCtlMgr: Exiting. Disconnecting from teamd. LAG 'PortChannel16'
Aug 25 14:52:44.346251 sonic NOTICE teamd#tlm_teamd: :- ~TeamdCtlMgr: Exiting. Disconnecting from teamd. LAG 'PortChannel23'
Aug 25 14:52:44.346530 sonic NOTICE teamd#tlm_teamd: :- ~TeamdCtlMgr: Exiting. Disconnecting from teamd. LAG 'PortChannel25'
Aug 25 14:52:44.346545 sonic NOTICE teamd#tlm_teamd: :- ~TeamdCtlMgr: Exiting. Disconnecting from teamd. LAG 'PortChannel11'
Aug 25 14:52:44.346564 sonic NOTICE teamd#tlm_teamd: :- ~TeamdCtlMgr: Exiting. Disconnecting from teamd. LAG 'PortChannel60'
Aug 25 14:52:44.346711 sonic NOTICE teamd#tlm_teamd: :- ~TeamdCtlMgr: Exiting. Disconnecting from teamd. LAG 'PortChannel29'
Aug 25 14:52:44.346742 sonic NOTICE teamd#tlm_teamd: :- ~TeamdCtlMgr: Exiting. Disconnecting from teamd. LAG 'PortChannel12'
Aug 25 14:52:44.346812 sonic NOTICE teamd#tlm_teamd: :- ~TeamdCtlMgr: Exiting. Disconnecting from teamd. LAG 'PortChannel24'
Aug 25 14:52:44.346823 sonic NOTICE teamd#tlm_teamd: :- ~TeamdCtlMgr: Exiting. Disconnecting from teamd. LAG 'PortChannel15'
Aug 25 14:52:44.346898 sonic NOTICE teamd#tlm_teamd: :- ~TeamdCtlMgr: Exiting. Disconnecting from teamd. LAG 'PortChannel30'
Aug 25 14:52:44.346934 sonic NOTICE teamd#tlm_teamd: :- ~TeamdCtlMgr: Exiting. Disconnecting from teamd. LAG 'PortChannel26'
Aug 25 14:52:44.347000 sonic NOTICE teamd#tlm_teamd: :- ~TeamdCtlMgr: Exiting. Disconnecting from teamd. LAG 'PortChannel27'
Aug 25 14:52:44.347023 sonic NOTICE teamd#tlm_teamd: :- ~TeamdCtlMgr: Exiting. Disconnecting from teamd. LAG 'PortChannel28'
Aug 25 14:52:44.347243 sonic NOTICE teamd#tlm_teamd: :- ~TeamdCtlMgr: Exiting. Disconnecting from teamd. LAG 'PortChannel39'
Aug 25 14:52:44.347262 sonic NOTICE teamd#tlm_teamd: :- ~TeamdCtlMgr: Exiting. Disconnecting from teamd. LAG 'PortChannel3'
Aug 25 14:52:44.347329 sonic NOTICE teamd#tlm_teamd: :- ~TeamdCtlMgr: Exiting. Disconnecting from teamd. LAG 'PortChannel21'
Aug 25 14:52:44.347342 sonic NOTICE teamd#tlm_teamd: :- ~TeamdCtlMgr: Exiting. Disconnecting from teamd. LAG 'PortChannel1'
Aug 25 14:52:44.347342 sonic NOTICE teamd#tlm_teamd: :- ~TeamdCtlMgr: Exiting. Disconnecting from teamd. LAG 'PortChannel34'
Aug 25 14:52:44.347447 sonic NOTICE teamd#tlm_teamd: :- ~TeamdCtlMgr: Exiting. Disconnecting from teamd. LAG 'PortChannel61'
Aug 25 14:52:44.347478 sonic NOTICE teamd#tlm_teamd: :- ~TeamdCtlMgr: Exiting. Disconnecting from teamd. LAG 'PortChannel38'
Aug 25 14:52:44.347478 sonic NOTICE teamd#tlm_teamd: :- ~TeamdCtlMgr: Exiting. Disconnecting from teamd. LAG 'PortChannel14'
Aug 25 14:52:44.347579 sonic NOTICE teamd#tlm_teamd: :- ~TeamdCtlMgr: Exiting. Disconnecting from teamd. LAG 'PortChannel31'
Aug 25 14:52:44.347669 sonic NOTICE teamd#tlm_teamd: :- ~TeamdCtlMgr: Exiting. Disconnecting from teamd. LAG 'PortChannel7'
Aug 25 14:52:44.347749 sonic NOTICE teamd#tlm_teamd: :- ~TeamdCtlMgr: Exiting. Disconnecting from teamd. LAG 'PortChannel33'
Aug 25 14:52:44.347766 sonic NOTICE teamd#tlm_teamd: :- ~TeamdCtlMgr: Exiting. Disconnecting from teamd. LAG 'PortChannel51'
Aug 25 14:52:44.347826 sonic NOTICE teamd#tlm_teamd: :- ~TeamdCtlMgr: Exiting. Disconnecting from teamd. LAG 'PortChannel35'
Aug 25 14:52:44.347850 sonic NOTICE teamd#tlm_teamd: :- ~TeamdCtlMgr: Exiting. Disconnecting from teamd. LAG 'PortChannel4'
Aug 25 14:52:44.347926 sonic NOTICE teamd#tlm_teamd: :- ~TeamdCtlMgr: Exiting. Disconnecting from teamd. LAG 'PortChannel36'
Aug 25 14:52:44.347926 sonic NOTICE teamd#tlm_teamd: :- ~TeamdCtlMgr: Exiting. Disconnecting from teamd. LAG 'PortChannel40'
Aug 25 14:52:44.498344 sonic NOTICE systemd[1]: /lib/systemd/system/kdump-tools.service:8: Standard output type syslog+console is obsolete, automatically updating to journal+console. Please update your unit file, and consider removing the setting altogether.
Aug 25 14:52:44.506888 sonic WARNING systemd[1]: Configuration file /lib/systemd/system/copp-config.service is marked executable. Please remove executable permission bits. Proceeding anyway.
Aug 25 14:52:44.619562 sonic INFO /hostcfgd: Feature lldp is enabled and started
Aug 25 14:52:44.648311 sonic INFO systemd[1]: Reloading.
Aug 25 14:52:44.918355 sonic NOTICE systemd[1]: /lib/systemd/system/kdump-tools.service:8: Standard output type syslog+console is obsolete, automatically updating to journal+console. Please update your unit file, and consider removing the setting altogether.
Aug 25 14:52:44.928070 sonic WARNING systemd[1]: Configuration file /lib/systemd/system/copp-config.service is marked executable. Please remove executable permission bits. Proceeding anyway.
Aug 25 14:52:45.071199 sonic INFO /hostcfgd: Feature macsec is stopped and disabled
Aug 25 14:52:45.096482 sonic INFO systemd[1]: Reloading.
Aug 25 14:52:45.119370 sonic INFO teamd#supervisord 2021-08-25 14:52:45,118 INFO stopped: tlm_teamd (exit status 0)
Aug 25 14:52:45.348856 sonic NOTICE systemd[1]: /lib/systemd/system/kdump-tools.service:8: Standard output type syslog+console is obsolete, automatically updating to journal+console. Please update your unit file, and consider removing the setting altogether.
Aug 25 14:52:45.358526 sonic WARNING systemd[1]: Configuration file /lib/systemd/system/copp-config.service is marked executable. Please remove executable permission bits. Proceeding anyway.
Aug 25 14:52:45.484829 sonic INFO /hostcfgd: Feature mgmt-framework is enabled and started
Aug 25 14:52:45.518849 sonic INFO systemd[1]: Reloading.
Aug 25 14:52:45.775400 sonic NOTICE systemd[1]: /lib/systemd/system/kdump-tools.service:8: Standard output type syslog+console is obsolete, automatically updating to journal+console. Please update your unit file, and consider removing the setting altogether.
Aug 25 14:52:45.783991 sonic WARNING systemd[1]: Configuration file /lib/systemd/system/copp-config.service is marked executable. Please remove executable permission bits. Proceeding anyway.
Aug 25 14:52:45.906910 sonic INFO /hostcfgd: Feature nat is stopped and disabled
Aug 25 14:52:45.934229 sonic INFO systemd[1]: Reloading.
Aug 25 14:52:46.120549 sonic NOTICE teamd#teamsyncd: :- cleanTeamSync: Cleaning up LAG teamd resources ...
Aug 25 14:52:46.133282 sonic NOTICE teamd#teamsyncd: :- main: Received SIGTERM Exiting
Aug 25 14:52:46.151183 sonic INFO teamd#supervisord 2021-08-25 14:52:46,150 INFO stopped: teamsyncd (exit status 0)
Aug 25 14:52:46.203787 sonic NOTICE systemd[1]: /lib/systemd/system/kdump-tools.service:8: Standard output type syslog+console is obsolete, automatically updating to journal+console. Please update your unit file, and consider removing the setting altogether.
Aug 25 14:52:46.211405 sonic WARNING systemd[1]: Configuration file /lib/systemd/system/copp-config.service is marked executable. Please remove executable permission bits. Proceeding anyway.
Aug 25 14:52:46.345388 sonic INFO /hostcfgd: Feature pmon is enabled and started
Aug 25 14:52:46.362105 sonic INFO systemd[1]: Reloading.
Aug 25 14:52:46.649655 sonic NOTICE systemd[1]: /lib/systemd/system/kdump-tools.service:8: Standard output type syslog+console is obsolete, automatically updating to journal+console. Please update your unit file, and consider removing the setting altogether.
Aug 25 14:52:46.659039 sonic WARNING systemd[1]: Configuration file /lib/systemd/system/copp-config.service is marked executable. Please remove executable permission bits. Proceeding anyway.
Aug 25 14:52:46.805923 sonic INFO /hostcfgd: Feature radv is enabled and started
Aug 25 14:52:46.829848 sonic INFO systemd[1]: Reloading.
Aug 25 14:52:47.075631 sonic NOTICE systemd[1]: /lib/systemd/system/kdump-tools.service:8: Standard output type syslog+console is obsolete, automatically updating to journal+console. Please update your unit file, and consider removing the setting altogether.
Aug 25 14:52:47.084170 sonic WARNING systemd[1]: Configuration file /lib/systemd/system/copp-config.service is marked executable. Please remove executable permission bits. Proceeding anyway.
Aug 25 14:52:47.152136 sonic NOTICE teamd#teammgrd: :- cleanTeamProcesses: Cleaning up LAGs during shutdown...
Aug 25 14:52:47.152136 sonic NOTICE teamd#teammgrd: :- cleanTeamProcesses: Patch by judyjoseph - aggregate remove approach
Aug 25 14:52:47.152136 sonic NOTICE teamd#teammgrd: :- removeLag: Stop port channel PortChannel1
Aug 25 14:52:47.152136 sonic NOTICE teamd#teammgrd: :- removeLag: Stop port channel PortChannel10
Aug 25 14:52:47.152136 sonic NOTICE teamd#teammgrd: :- removeLag: Stop port channel PortChannel11
Aug 25 14:52:47.152136 sonic NOTICE teamd#teammgrd: :- removeLag: Stop port channel PortChannel12
Aug 25 14:52:47.152203 sonic NOTICE teamd#teammgrd: :- removeLag: Stop port channel PortChannel13
Aug 25 14:52:47.152203 sonic NOTICE teamd#teammgrd: :- removeLag: Stop port channel PortChannel14
Aug 25 14:52:47.152203 sonic NOTICE teamd#teammgrd: :- removeLag: Stop port channel PortChannel15
Aug 25 14:52:47.152203 sonic NOTICE teamd#teammgrd: :- removeLag: Stop port channel PortChannel16
Aug 25 14:52:47.152249 sonic NOTICE teamd#teammgrd: :- removeLag: Stop port channel PortChannel17
Aug 25 14:52:47.152249 sonic NOTICE teamd#teammgrd: :- removeLag: Stop port channel PortChannel18
Aug 25 14:52:47.152249 sonic NOTICE teamd#teammgrd: :- removeLag: Stop port channel PortChannel19
Aug 25 14:52:47.152281 sonic NOTICE teamd#teammgrd: :- removeLag: Stop port channel PortChannel2
Aug 25 14:52:47.152330 sonic NOTICE teamd#teammgrd: :- removeLag: Stop port channel PortChannel20
Aug 25 14:52:47.152330 sonic NOTICE teamd#teammgrd: :- removeLag: Stop port channel PortChannel21
Aug 25 14:52:47.152330 sonic NOTICE teamd#teammgrd: :- removeLag: Stop port channel PortChannel22
Aug 25 14:52:47.152356 sonic NOTICE teamd#teammgrd: :- removeLag: Stop port channel PortChannel23
Aug 25 14:52:47.152356 sonic NOTICE teamd#teammgrd: :- removeLag: Stop port channel PortChannel24
Aug 25 14:52:47.152374 sonic NOTICE teamd#teammgrd: :- removeLag: Stop port channel PortChannel25
Aug 25 14:52:47.152480 sonic NOTICE teamd#teammgrd: :- removeLag: Stop port channel PortChannel26
Aug 25 14:52:47.152480 sonic NOTICE teamd#teammgrd: :- removeLag: Stop port channel PortChannel27
Aug 25 14:52:47.152501 sonic NOTICE teamd#teammgrd: :- removeLag: Stop port channel PortChannel28
Aug 25 14:52:47.152542 sonic NOTICE teamd#teammgrd: :- removeLag: Stop port channel PortChannel29
Aug 25 14:52:47.152599 sonic NOTICE teamd#teammgrd: :- removeLag: Stop port channel PortChannel3
Aug 25 14:52:47.152599 sonic NOTICE teamd#teammgrd: :- removeLag: Stop port channel PortChannel30
Aug 25 14:52:47.152599 sonic NOTICE teamd#teammgrd: :- removeLag: Stop port channel PortChannel31
Aug 25 14:52:47.152599 sonic NOTICE teamd#teammgrd: :- removeLag: Stop port channel PortChannel32
Aug 25 14:52:47.152599 sonic NOTICE teamd#teammgrd: :- removeLag: Stop port channel PortChannel33
Aug 25 14:52:47.152599 sonic NOTICE teamd#teammgrd: :- removeLag: Stop port channel PortChannel34
Aug 25 14:52:47.152599 sonic NOTICE teamd#teammgrd: :- removeLag: Stop port channel PortChannel35
Aug 25 14:52:47.152676 sonic NOTICE teamd#teammgrd: :- removeLag: Stop port channel PortChannel36
Aug 25 14:52:47.152676 sonic NOTICE teamd#teammgrd: :- removeLag: Stop port channel PortChannel37
Aug 25 14:52:47.152676 sonic NOTICE teamd#teammgrd: :- removeLag: Stop port channel PortChannel38
Aug 25 14:52:47.152676 sonic NOTICE teamd#teammgrd: :- removeLag: Stop port channel PortChannel39
Aug 25 14:52:47.152676 sonic NOTICE teamd#teammgrd: :- removeLag: Stop port channel PortChannel4
Aug 25 14:52:47.152676 sonic NOTICE teamd#teammgrd: :- removeLag: Stop port channel PortChannel40
Aug 25 14:52:47.152676 sonic NOTICE teamd#teammgrd: :- removeLag: Stop port channel PortChannel41
Aug 25 14:52:47.152734 sonic NOTICE teamd#teammgrd: :- removeLag: Stop port channel PortChannel42
Aug 25 14:52:47.152734 sonic NOTICE teamd#teammgrd: :- removeLag: Stop port channel PortChannel43
Aug 25 14:52:47.152734 sonic NOTICE teamd#teammgrd: :- removeLag: Stop port channel PortChannel44
Aug 25 14:52:47.152734 sonic NOTICE teamd#teammgrd: :- removeLag: Stop port channel PortChannel45
Aug 25 14:52:47.152734 sonic NOTICE teamd#teammgrd: :- removeLag: Stop port channel PortChannel46
Aug 25 14:52:47.152734 sonic NOTICE teamd#teammgrd: :- removeLag: Stop port channel PortChannel47
Aug 25 14:52:47.152734 sonic NOTICE teamd#teammgrd: :- removeLag: Stop port channel PortChannel48
Aug 25 14:52:47.152734 sonic NOTICE teamd#teammgrd: :- removeLag: Stop port channel PortChannel49
Aug 25 14:52:47.152839 sonic NOTICE teamd#teammgrd: :- removeLag: Stop port channel PortChannel5
Aug 25 14:52:47.152839 sonic NOTICE teamd#teammgrd: :- removeLag: Stop port channel PortChannel50
Aug 25 14:52:47.152839 sonic NOTICE teamd#teammgrd: :- removeLag: Stop port channel PortChannel51
Aug 25 14:52:47.152839 sonic NOTICE teamd#teammgrd: :- removeLag: Stop port channel PortChannel52
Aug 25 14:52:47.152839 sonic NOTICE teamd#teammgrd: :- removeLag: Stop port channel PortChannel53
Aug 25 14:52:47.152839 sonic NOTICE teamd#teammgrd: :- removeLag: Stop port channel PortChannel54
Aug 25 14:52:47.152839 sonic NOTICE teamd#teammgrd: :- removeLag: Stop port channel PortChannel55
Aug 25 14:52:47.152878 sonic NOTICE teamd#teammgrd: :- removeLag: Stop port channel PortChannel56
Aug 25 14:52:47.152878 sonic NOTICE teamd#teammgrd: :- removeLag: Stop port channel PortChannel57
Aug 25 14:52:47.152917 sonic NOTICE teamd#teammgrd: :- removeLag: Stop port channel PortChannel58
Aug 25 14:52:47.152917 sonic NOTICE teamd#teammgrd: :- removeLag: Stop port channel PortChannel59
Aug 25 14:52:47.152959 sonic NOTICE teamd#teammgrd: :- removeLag: Stop port channel PortChannel6
Aug 25 14:52:47.153000 sonic NOTICE teamd#teammgrd: :- removeLag: Stop port channel PortChannel60
Aug 25 14:52:47.153024 sonic NOTICE teamd#teammgrd: :- removeLag: Stop port channel PortChannel61
Aug 25 14:52:47.153061 sonic NOTICE teamd#teammgrd: :- removeLag: Stop port channel PortChannel62
Aug 25 14:52:47.153061 sonic NOTICE teamd#teammgrd: :- removeLag: Stop port channel PortChannel63
Aug 25 14:52:47.153061 sonic INFO teamd#supervisord 2021-08-25 14:52:47,152 INFO waiting for supervisor-proc-exit-listener, rsyslogd, teammgrd to die
Aug 25 14:52:47.153061 sonic NOTICE teamd#teammgrd: :- removeLag: Stop port channel PortChannel64
Aug 25 14:52:47.153061 sonic NOTICE teamd#teammgrd: :- removeLag: Stop port channel PortChannel7
Aug 25 14:52:47.153106 sonic NOTICE teamd#teammgrd: :- removeLag: Stop port channel PortChannel8
Aug 25 14:52:47.153106 sonic NOTICE teamd#teammgrd: :- removeLag: Stop port channel PortChannel9
Aug 25 14:52:47.163809 sonic NOTICE swss#portsyncd: :- onMsg: nlmsg type:16 key:PortChannel1 admin:0 oper:0 addr:1c:34:da:1d:42:00 ifindex:410 master:0 type:team
Aug 25 14:52:47.166245 sonic NOTICE swss#portsyncd: :- onMsg: nlmsg type:17 key:PortChannel1 admin:0 oper:0 addr:1c:34:da:1d:42:00 ifindex:410 master:0 type:team
Aug 25 14:52:47.205750 sonic INFO /hostcfgd: Running cmd: 'sudo systemctl unmask restapi.service'
Aug 25 14:52:47.235295 sonic INFO hostcfgd[57178]: Unit restapi.service does not exist, proceeding anyway.
Aug 25 14:52:47.236912 sonic INFO systemd[1]: Reloading.
Aug 25 14:52:47.542153 sonic NOTICE systemd[1]: /lib/systemd/system/kdump-tools.service:8: Standard output type syslog+console is obsolete, automatically updating to journal+console. Please update your unit file, and consider removing the setting altogether.
Aug 25 14:52:47.552664 sonic WARNING systemd[1]: Configuration file /lib/systemd/system/copp-config.service is marked executable. Please remove executable permission bits. Proceeding anyway.
Aug 25 14:52:47.664844 sonic INFO /hostcfgd: Running cmd: 'sudo systemctl enable restapi.service'
Aug 25 14:52:47.696038 sonic INFO hostcfgd[57225]: Failed to enable unit: Unit file restapi.service does not exist.
Aug 25 14:52:47.697066 sonic ERR /hostcfgd: sudo systemctl enable restapi.service - failed: return code - 1, output:#012None
Aug 25 14:52:47.697216 sonic ERR /hostcfgd: Feature 'restapi.service' failed to be enabled and started
Aug 25 14:52:47.697345 sonic INFO /hostcfgd: Feature restapi is enabled and started
Aug 25 14:52:47.725765 sonic INFO systemd[1]: Reloading.
Aug 25 14:52:47.995040 sonic NOTICE systemd[1]: /lib/systemd/system/kdump-tools.service:8: Standard output type syslog+console is obsolete, automatically updating to journal+console. Please update your unit file, and consider removing the setting altogether.
Aug 25 14:52:48.004749 sonic WARNING systemd[1]: Configuration file /lib/systemd/system/copp-config.service is marked executable. Please remove executable permission bits. Proceeding anyway.
Aug 25 14:52:48.132671 sonic INFO /hostcfgd: Feature sflow is stopped and disabled
Aug 25 14:52:48.154680 sonic INFO teamd#supervisord 2021-08-25 14:52:48,153 INFO reaped unknown pid 26 (exit status 0)
Aug 25 14:52:48.160570 sonic INFO systemd[1]: Reloading.
Aug 25 14:52:48.178611 sonic NOTICE swss#portsyncd: :- onMsg: nlmsg type:16 key:PortChannel10 admin:0 oper:0 addr:1c:34:da:1d:42:00 ifindex:411 master:0 type:team
Aug 25 14:52:48.183269 sonic NOTICE swss#portsyncd: :- onMsg: nlmsg type:17 key:PortChannel10 admin:0 oper:0 addr:1c:34:da:1d:42:00 ifindex:411 master:0 type:team
Aug 25 14:52:48.428975 sonic NOTICE systemd[1]: /lib/systemd/system/kdump-tools.service:8: Standard output type syslog+console is obsolete, automatically updating to journal+console. Please update your unit file, and consider removing the setting altogether.
Aug 25 14:52:48.438468 sonic WARNING systemd[1]: Configuration file /lib/systemd/system/copp-config.service is marked executable. Please remove executable permission bits. Proceeding anyway.
Aug 25 14:52:48.568345 sonic INFO /hostcfgd: Feature snmp is enabled and started
Aug 25 14:52:48.596027 sonic INFO systemd[1]: Reloading.
Aug 25 14:52:48.863995 sonic NOTICE systemd[1]: /lib/systemd/system/kdump-tools.service:8: Standard output type syslog+console is obsolete, automatically updating to journal+console. Please update your unit file, and consider removing the setting altogether.
Aug 25 14:52:48.873834 sonic WARNING systemd[1]: Configuration file /lib/systemd/system/copp-config.service is marked executable. Please remove executable permission bits. Proceeding anyway.
Aug 25 14:52:49.031973 sonic INFO /hostcfgd: Feature swss is enabled and started
Aug 25 14:52:49.054100 sonic INFO systemd[1]: Reloading.
Aug 25 14:52:49.156523 sonic INFO teamd#supervisord 2021-08-25 14:52:49,156 INFO reaped unknown pid 34 (exit status 0)
Aug 25 14:52:49.193518 sonic NOTICE swss#portsyncd: :- onMsg: nlmsg type:16 key:PortChannel11 admin:0 oper:0 addr:1c:34:da:1d:42:00 ifindex:412 master:0 type:team
Aug 25 14:52:49.196536 sonic NOTICE swss#portsyncd: :- onMsg: nlmsg type:17 key:PortChannel11 admin:0 oper:0 addr:1c:34:da:1d:42:00 ifindex:412 master:0 type:team
Aug 25 14:52:49.354551 sonic NOTICE systemd[1]: /lib/systemd/system/kdump-tools.service:8: Standard output type syslog+console is obsolete, automatically updating to journal+console. Please update your unit file, and consider removing the setting altogether.
Aug 25 14:52:49.364267 sonic WARNING systemd[1]: Configuration file /lib/systemd/system/copp-config.service is marked executable. Please remove executable permission bits. Proceeding anyway.
Aug 25 14:52:49.497709 sonic INFO /hostcfgd: Feature syncd is enabled and started
Aug 25 14:52:49.520374 sonic INFO systemd[1]: Reloading.
Aug 25 14:52:49.801043 sonic NOTICE systemd[1]: /lib/systemd/system/kdump-tools.service:8: Standard output type syslog+console is obsolete, automatically updating to journal+console. Please update your unit file, and consider removing the setting altogether.
Aug 25 14:52:49.811673 sonic WARNING systemd[1]: Configuration file /lib/systemd/system/copp-config.service is marked executable. Please remove executable permission bits. Proceeding anyway.
Aug 25 14:52:49.972701 sonic INFO /hostcfgd: Feature teamd is enabled and started
Aug 25 14:52:50.023659 sonic INFO systemd[1]: Reloading.
Aug 25 14:52:50.158506 sonic INFO teamd#supervisord 2021-08-25 14:52:50,157 INFO reaped unknown pid 42 (exit status 0)
Aug 25 14:52:50.158642 sonic INFO teamd#supervisord 2021-08-25 14:52:50,158 INFO waiting for supervisor-proc-exit-listener, rsyslogd, teammgrd to die
Aug 25 14:52:50.197557 sonic NOTICE swss#portsyncd: :- onMsg: nlmsg type:16 key:PortChannel12 admin:0 oper:0 addr:1c:34:da:1d:42:00 ifindex:413 master:0 type:team
Aug 25 14:52:50.201527 sonic NOTICE swss#portsyncd: :- onMsg: nlmsg type:17 key:PortChannel12 admin:0 oper:0 addr:1c:34:da:1d:42:00 ifindex:413 master:0 type:team
Aug 25 14:52:50.299266 sonic NOTICE systemd[1]: /lib/systemd/system/kdump-tools.service:8: Standard output type syslog+console is obsolete, automatically updating to journal+console. Please update your unit file, and consider removing the setting altogether.
Aug 25 14:52:50.309058 sonic WARNING systemd[1]: Configuration file /lib/systemd/system/copp-config.service is marked executable. Please remove executable permission bits. Proceeding anyway.
Aug 25 14:52:50.461056 sonic INFO /hostcfgd: Feature telemetry is enabled and started
Aug 25 14:52:50.488768 sonic INFO systemd[1]: Reloading.
Aug 25 14:52:50.765029 sonic NOTICE systemd[1]: /lib/systemd/system/kdump-tools.service:8: Standard output type syslog+console is obsolete, automatically updating to journal+console. Please update your unit file, and consider removing the setting altogether.
Aug 25 14:52:50.774026 sonic WARNING systemd[1]: Configuration file /lib/systemd/system/copp-config.service is marked executable. Please remove executable permission bits. Proceeding anyway.
Aug 25 14:52:50.936835 sonic INFO /hostcfgd: Feature what-just-happened is stopped and disabled
Aug 25 14:52:51.020472 sonic INFO /hostcfgd: cmd - service aaastatsd stop
Aug 25 14:52:51.088982 sonic WARNING systemd[1]: Configuration file /lib/systemd/system/copp-config.service is marked executable. Please remove executable permission bits. Proceeding anyway.
Aug 25 14:52:51.160873 sonic INFO teamd#supervisord 2021-08-25 14:52:51,160 INFO reaped unknown pid 50 (exit status 0)
Aug 25 14:52:51.210063 sonic NOTICE swss#portsyncd: :- onMsg: nlmsg type:16 key:PortChannel13 admin:0 oper:0 addr:1c:34:da:1d:42:00 ifindex:414 master:0 type:team
Aug 25 14:52:51.212082 sonic NOTICE swss#portsyncd: :- onMsg: nlmsg type:17 key:PortChannel13 admin:0 oper:0 addr:1c:34:da:1d:42:00 ifindex:414 master:0 type:team
Aug 25 14:52:51.904017 sonic INFO systemd[1]: Stopping AAA Statistics Collection daemon...
Aug 25 14:52:51.910198 sonic INFO systemd[1]: aaastatsd.service: Succeeded.
Aug 25 14:52:51.911184 sonic INFO systemd[1]: Stopped AAA Statistics Collection daemon.
Aug 25 14:52:51.912442 sonic INFO systemd[1]: aaastatsd.timer: Succeeded.
Aug 25 14:52:51.912714 sonic INFO systemd[1]: Stopped Delays aaastatsd daemon until SONiC has started.
Aug 25 14:52:51.915193 sonic INFO /hostcfgd: NtpCfg load ...
Aug 25 14:52:51.915347 sonic INFO /hostcfgd: ntp server update key 0 data {}
Aug 25 14:52:51.915483 sonic INFO /hostcfgd: ntp server update, restarting ntp-config, ntp server exists False
Aug 25 14:52:51.923075 sonic INFO systemd[1]: ntp-config.service: Succeeded.
Aug 25 14:52:51.924179 sonic INFO systemd[1]: Stopped Update NTP configuration.
Aug 25 14:52:51.924484 sonic INFO systemd[1]: Stopping Update NTP configuration...
Aug 25 14:52:52.162687 sonic INFO teamd#supervisord 2021-08-25 14:52:52,161 INFO reaped unknown pid 58 (exit status 0)
Aug 25 14:52:52.224096 sonic NOTICE swss#portsyncd: :- onMsg: nlmsg type:16 key:PortChannel14 admin:0 oper:0 addr:1c:34:da:1d:42:00 ifindex:415 master:0 type:team
Aug 25 14:52:52.227377 sonic NOTICE swss#portsyncd: :- onMsg: nlmsg type:17 key:PortChannel14 admin:0 oper:0 addr:1c:34:da:1d:42:00 ifindex:415 master:0 type:team
Aug 25 14:52:53.165429 sonic INFO teamd#supervisord 2021-08-25 14:52:53,163 INFO reaped unknown pid 66 (exit status 0)
Aug 25 14:52:53.165429 sonic INFO teamd#supervisord 2021-08-25 14:52:53,164 INFO waiting for supervisor-proc-exit-listener, rsyslogd, teammgrd to die
Aug 25 14:52:53.237513 sonic NOTICE swss#portsyncd: :- onMsg: nlmsg type:16 key:PortChannel15 admin:0 oper:0 addr:1c:34:da:1d:42:00 ifindex:416 master:0 type:team
Aug 25 14:52:53.240298 sonic NOTICE swss#portsyncd: :- onMsg: nlmsg type:17 key:PortChannel15 admin:0 oper:0 addr:1c:34:da:1d:42:00 ifindex:416 master:0 type:team
Aug 25 14:52:54.166574 sonic INFO teamd#supervisord 2021-08-25 14:52:54,165 INFO reaped unknown pid 74 (exit status 0)
Aug 25 14:52:54.262886 sonic NOTICE swss#portsyncd: :- onMsg: nlmsg type:16 key:PortChannel16 admin:0 oper:0 addr:1c:34:da:1d:42:00 ifindex:417 master:0 type:team
Aug 25 14:52:54.266435 sonic NOTICE swss#portsyncd: :- onMsg: nlmsg type:17 key:PortChannel16 admin:0 oper:0 addr:1c:34:da:1d:42:00 ifindex:417 master:0 type:team
Aug 25 14:52:55.168298 sonic INFO teamd#supervisord 2021-08-25 14:52:55,167 INFO reaped unknown pid 82 (exit status 0)
Aug 25 14:52:55.271515 sonic NOTICE swss#portsyncd: :- onMsg: nlmsg type:16 key:PortChannel17 admin:0 oper:0 addr:1c:34:da:1d:42:00 ifindex:418 master:0 type:team
Aug 25 14:52:55.274994 sonic NOTICE swss#portsyncd: :- onMsg: nlmsg type:17 key:PortChannel17 admin:0 oper:0 addr:1c:34:da:1d:42:00 ifindex:418 master:0 type:team
Aug 25 14:52:55.343932 sonic WARNING pmon#xcvrd: Could not retreive port inside config_db PORT table
Aug 25 14:52:55.374938 sonic INFO pmon#/supervisord: xcvrd Aug 25 14:52:55 INFO    LOG: Initializing SX log with STDOUT as output file.
Aug 25 14:52:56.171758 sonic INFO teamd#supervisord 2021-08-25 14:52:56,169 INFO reaped unknown pid 90 (exit status 0)
Aug 25 14:52:56.171758 sonic INFO teamd#supervisord 2021-08-25 14:52:56,169 INFO waiting for supervisor-proc-exit-listener, rsyslogd, teammgrd to die
Aug 25 14:52:56.269451 sonic NOTICE swss#portsyncd: :- onMsg: nlmsg type:16 key:PortChannel18 admin:0 oper:0 addr:1c:34:da:1d:42:00 ifindex:419 master:0 type:team
Aug 25 14:52:56.273468 sonic NOTICE swss#portsyncd: :- onMsg: nlmsg type:17 key:PortChannel18 admin:0 oper:0 addr:1c:34:da:1d:42:00 ifindex:419 master:0 type:team
Aug 25 14:52:57.172730 sonic INFO teamd#supervisord 2021-08-25 14:52:57,172 INFO reaped unknown pid 98 (exit status 0)
Aug 25 14:52:57.283868 sonic NOTICE swss#portsyncd: :- onMsg: nlmsg type:16 key:PortChannel19 admin:0 oper:0 addr:1c:34:da:1d:42:00 ifindex:420 master:0 type:team
Aug 25 14:52:57.285157 sonic NOTICE swss#portsyncd: :- onMsg: nlmsg type:17 key:PortChannel19 admin:0 oper:0 addr:1c:34:da:1d:42:00 ifindex:420 master:0 type:team
Aug 25 14:52:58.174518 sonic INFO teamd#supervisord 2021-08-25 14:52:58,173 INFO reaped unknown pid 106 (exit status 0)
Aug 25 14:52:58.196048 sonic NOTICE swss#portsyncd: :- onMsg: nlmsg type:16 key:PortChannel2 admin:0 oper:0 addr:1c:34:da:1d:42:00 ifindex:421 master:0 type:team
Aug 25 14:52:58.198249 sonic NOTICE swss#portsyncd: :- onMsg: nlmsg type:17 key:PortChannel2 admin:0 oper:0 addr:1c:34:da:1d:42:00 ifindex:421 master:0 type:team
Aug 25 14:52:59.177893 sonic INFO teamd#supervisord 2021-08-25 14:52:59,176 INFO reaped unknown pid 114 (exit status 0)
Aug 25 14:52:59.177893 sonic INFO teamd#supervisord 2021-08-25 14:52:59,176 INFO waiting for supervisor-proc-exit-listener, rsyslogd, teammgrd to die
Aug 25 14:52:59.207992 sonic NOTICE swss#portsyncd: :- onMsg: nlmsg type:16 key:PortChannel20 admin:0 oper:0 addr:1c:34:da:1d:42:00 ifindex:422 master:0 type:team
Aug 25 14:52:59.211044 sonic NOTICE swss#portsyncd: :- onMsg: nlmsg type:17 key:PortChannel20 admin:0 oper:0 addr:1c:34:da:1d:42:00 ifindex:422 master:0 type:team
Aug 25 14:53:00.178805 sonic INFO teamd#supervisord 2021-08-25 14:53:00,178 INFO reaped unknown pid 122 (exit status 0)
Aug 25 14:53:00.221714 sonic NOTICE swss#portsyncd: :- onMsg: nlmsg type:16 key:PortChannel21 admin:0 oper:0 addr:1c:34:da:1d:42:00 ifindex:423 master:0 type:team
Aug 25 14:53:00.225443 sonic NOTICE swss#portsyncd: :- onMsg: nlmsg type:17 key:PortChannel21 admin:0 oper:0 addr:1c:34:da:1d:42:00 ifindex:423 master:0 type:team
Aug 25 14:53:01.181391 sonic INFO teamd#supervisord 2021-08-25 14:53:01,180 INFO reaped unknown pid 130 (exit status 0)
Aug 25 14:53:01.236497 sonic NOTICE swss#portsyncd: :- onMsg: nlmsg type:16 key:PortChannel22 admin:0 oper:0 addr:1c:34:da:1d:42:00 ifindex:424 master:0 type:team
Aug 25 14:53:01.238532 sonic NOTICE swss#portsyncd: :- onMsg: nlmsg type:17 key:PortChannel22 admin:0 oper:0 addr:1c:34:da:1d:42:00 ifindex:424 master:0 type:team
Aug 25 14:53:02.184787 sonic INFO teamd#supervisord 2021-08-25 14:53:02,183 INFO reaped unknown pid 138 (exit status 0)
Aug 25 14:53:02.184787 sonic INFO teamd#supervisord 2021-08-25 14:53:02,183 INFO waiting for supervisor-proc-exit-listener, rsyslogd, teammgrd to die
Aug 25 14:53:02.248083 sonic NOTICE swss#portsyncd: :- onMsg: nlmsg type:16 key:PortChannel23 admin:0 oper:0 addr:1c:34:da:1d:42:00 ifindex:425 master:0 type:team
Aug 25 14:53:02.250280 sonic NOTICE swss#portsyncd: :- onMsg: nlmsg type:17 key:PortChannel23 admin:0 oper:0 addr:1c:34:da:1d:42:00 ifindex:425 master:0 type:team
Aug 25 14:53:02.574905 sonic NOTICE syncd#SDK: [CORE_API.NOTICE] Initialize Bin Allocator delay: 56
Aug 25 14:53:03.186566 sonic INFO teamd#supervisord 2021-08-25 14:53:03,185 INFO reaped unknown pid 146 (exit status 0)
Aug 25 14:53:03.261877 sonic NOTICE swss#portsyncd: :- onMsg: nlmsg type:16 key:PortChannel24 admin:0 oper:0 addr:1c:34:da:1d:42:00 ifindex:426 master:0 type:team
Aug 25 14:53:03.263552 sonic NOTICE swss#portsyncd: :- onMsg: nlmsg type:17 key:PortChannel24 admin:0 oper:0 addr:1c:34:da:1d:42:00 ifindex:426 master:0 type:team
Aug 25 14:53:04.188922 sonic INFO teamd#supervisord 2021-08-25 14:53:04,188 INFO reaped unknown pid 154 (exit status 0)
Aug 25 14:53:04.275471 sonic NOTICE swss#portsyncd: :- onMsg: nlmsg type:16 key:PortChannel25 admin:0 oper:0 addr:1c:34:da:1d:42:00 ifindex:427 master:0 type:team
Aug 25 14:53:04.277250 sonic NOTICE swss#portsyncd: :- onMsg: nlmsg type:17 key:PortChannel25 admin:0 oper:0 addr:1c:34:da:1d:42:00 ifindex:427 master:0 type:team
Aug 25 14:53:05.192312 sonic INFO teamd#supervisord 2021-08-25 14:53:05,190 INFO reaped unknown pid 162 (exit status 0)
Aug 25 14:53:05.192312 sonic INFO teamd#supervisord 2021-08-25 14:53:05,191 INFO waiting for supervisor-proc-exit-listener, rsyslogd, teammgrd to die
Aug 25 14:53:05.287601 sonic NOTICE swss#portsyncd: :- onMsg: nlmsg type:16 key:PortChannel26 admin:0 oper:0 addr:1c:34:da:1d:42:00 ifindex:428 master:0 type:team
Aug 25 14:53:05.291012 sonic NOTICE swss#portsyncd: :- onMsg: nlmsg type:17 key:PortChannel26 admin:0 oper:0 addr:1c:34:da:1d:42:00 ifindex:428 master:0 type:team
Aug 25 14:53:06.195037 sonic INFO teamd#supervisord 2021-08-25 14:53:06,193 INFO reaped unknown pid 170 (exit status 0)
Aug 25 14:53:06.300723 sonic NOTICE swss#portsyncd: :- onMsg: nlmsg type:16 key:PortChannel27 admin:0 oper:0 addr:1c:34:da:1d:42:00 ifindex:429 master:0 type:team
Aug 25 14:53:06.303449 sonic NOTICE swss#portsyncd: :- onMsg: nlmsg type:17 key:PortChannel27 admin:0 oper:0 addr:1c:34:da:1d:42:00 ifindex:429 master:0 type:team
Aug 25 14:53:07.195952 sonic INFO teamd#supervisord 2021-08-25 14:53:07,195 INFO reaped unknown pid 178 (exit status 0)
Aug 25 14:53:07.206655 sonic NOTICE swss#portsyncd: :- onMsg: nlmsg type:16 key:PortChannel28 admin:0 oper:0 addr:1c:34:da:1d:42:00 ifindex:430 master:0 type:team
Aug 25 14:53:07.208060 sonic NOTICE swss#portsyncd: :- onMsg: nlmsg type:17 key:PortChannel28 admin:0 oper:0 addr:1c:34:da:1d:42:00 ifindex:430 master:0 type:team
Aug 25 14:53:08.197862 sonic INFO teamd#supervisord 2021-08-25 14:53:08,197 INFO reaped unknown pid 186 (exit status 0)
Aug 25 14:53:08.198642 sonic INFO teamd#supervisord 2021-08-25 14:53:08,197 INFO waiting for supervisor-proc-exit-listener, rsyslogd, teammgrd to die
Aug 25 14:53:08.220879 sonic NOTICE swss#portsyncd: :- onMsg: nlmsg type:16 key:PortChannel29 admin:0 oper:0 addr:1c:34:da:1d:42:00 ifindex:431 master:0 type:team
Aug 25 14:53:08.221998 sonic NOTICE swss#portsyncd: :- onMsg: nlmsg type:17 key:PortChannel29 admin:0 oper:0 addr:1c:34:da:1d:42:00 ifindex:431 master:0 type:team
Aug 25 14:53:09.200778 sonic INFO teamd#supervisord 2021-08-25 14:53:09,200 INFO reaped unknown pid 194 (exit status 0)
Aug 25 14:53:09.232434 sonic NOTICE swss#portsyncd: :- onMsg: nlmsg type:16 key:PortChannel3 admin:0 oper:0 addr:1c:34:da:1d:42:00 ifindex:432 master:0 type:team
Aug 25 14:53:09.235288 sonic NOTICE swss#portsyncd: :- onMsg: nlmsg type:17 key:PortChannel3 admin:0 oper:0 addr:1c:34:da:1d:42:00 ifindex:432 master:0 type:team
Aug 25 14:53:10.203074 sonic INFO teamd#supervisord 2021-08-25 14:53:10,202 INFO reaped unknown pid 202 (exit status 0)
Aug 25 14:53:10.245852 sonic NOTICE swss#portsyncd: :- onMsg: nlmsg type:16 key:PortChannel30 admin:0 oper:0 addr:1c:34:da:1d:42:00 ifindex:433 master:0 type:team
Aug 25 14:53:10.248150 sonic NOTICE swss#portsyncd: :- onMsg: nlmsg type:17 key:PortChannel30 admin:0 oper:0 addr:1c:34:da:1d:42:00 ifindex:433 master:0 type:team
Aug 25 14:53:10.634320 sonic WARNING pmon#thermalctld: Temperature of ASIC changed too fast, from 78.0 to 91.0, please check your hardware
Aug 25 14:53:11.206392 sonic INFO teamd#supervisord 2021-08-25 14:53:11,204 INFO reaped unknown pid 210 (exit status 0)
Aug 25 14:53:11.206392 sonic INFO teamd#supervisord 2021-08-25 14:53:11,205 INFO waiting for supervisor-proc-exit-listener, rsyslogd, teammgrd to die
Aug 25 14:53:11.258855 sonic NOTICE swss#portsyncd: :- onMsg: nlmsg type:16 key:PortChannel31 admin:0 oper:0 addr:1c:34:da:1d:42:00 ifindex:434 master:0 type:team
Aug 25 14:53:11.261849 sonic NOTICE swss#portsyncd: :- onMsg: nlmsg type:17 key:PortChannel31 admin:0 oper:0 addr:1c:34:da:1d:42:00 ifindex:434 master:0 type:team
Aug 25 14:53:12.208313 sonic INFO teamd#supervisord 2021-08-25 14:53:12,207 INFO reaped unknown pid 218 (exit status 0)
Aug 25 14:53:12.271014 sonic NOTICE swss#portsyncd: :- onMsg: nlmsg type:16 key:PortChannel32 admin:0 oper:0 addr:1c:34:da:1d:42:00 ifindex:435 master:0 type:team
Aug 25 14:53:12.273318 sonic NOTICE swss#portsyncd: :- onMsg: nlmsg type:17 key:PortChannel32 admin:0 oper:0 addr:1c:34:da:1d:42:00 ifindex:435 master:0 type:team
Aug 25 14:53:13.210130 sonic INFO teamd#supervisord 2021-08-25 14:53:13,209 INFO reaped unknown pid 226 (exit status 0)
Aug 25 14:53:13.283946 sonic NOTICE swss#portsyncd: :- onMsg: nlmsg type:16 key:PortChannel33 admin:0 oper:0 addr:1c:34:da:1d:42:00 ifindex:436 master:0 type:team
Aug 25 14:53:13.286410 sonic NOTICE swss#portsyncd: :- onMsg: nlmsg type:17 key:PortChannel33 admin:0 oper:0 addr:1c:34:da:1d:42:00 ifindex:436 master:0 type:team
Aug 25 14:53:14.212950 sonic INFO teamd#supervisord 2021-08-25 14:53:14,211 INFO reaped unknown pid 234 (exit status 0)
Aug 25 14:53:14.212950 sonic INFO teamd#supervisord 2021-08-25 14:53:14,211 INFO waiting for supervisor-proc-exit-listener, rsyslogd, teammgrd to die
Aug 25 14:53:14.296749 sonic NOTICE swss#portsyncd: :- onMsg: nlmsg type:16 key:PortChannel34 admin:0 oper:0 addr:1c:34:da:1d:42:00 ifindex:437 master:0 type:team
Aug 25 14:53:14.299714 sonic NOTICE swss#portsyncd: :- onMsg: nlmsg type:17 key:PortChannel34 admin:0 oper:0 addr:1c:34:da:1d:42:00 ifindex:437 master:0 type:team
Aug 25 14:53:15.214800 sonic INFO teamd#supervisord 2021-08-25 14:53:15,214 INFO reaped unknown pid 242 (exit status 0)
Aug 25 14:53:15.310268 sonic NOTICE swss#portsyncd: :- onMsg: nlmsg type:16 key:PortChannel35 admin:0 oper:0 addr:1c:34:da:1d:42:00 ifindex:438 master:0 type:team
Aug 25 14:53:15.311263 sonic NOTICE swss#portsyncd: :- onMsg: nlmsg type:17 key:PortChannel35 admin:0 oper:0 addr:1c:34:da:1d:42:00 ifindex:438 master:0 type:team
Aug 25 14:53:16.217456 sonic INFO teamd#supervisord 2021-08-25 14:53:16,215 INFO reaped unknown pid 250 (exit status 0)
Aug 25 14:53:16.323463 sonic NOTICE swss#portsyncd: :- onMsg: nlmsg type:16 key:PortChannel36 admin:0 oper:0 addr:1c:34:da:1d:42:00 ifindex:439 master:0 type:team
Aug 25 14:53:16.325837 sonic NOTICE swss#portsyncd: :- onMsg: nlmsg type:17 key:PortChannel36 admin:0 oper:0 addr:1c:34:da:1d:42:00 ifindex:439 master:0 type:team
Aug 25 14:53:17.219936 sonic INFO teamd#supervisord 2021-08-25 14:53:17,218 INFO reaped unknown pid 258 (exit status 0)
Aug 25 14:53:17.219936 sonic INFO teamd#supervisord 2021-08-25 14:53:17,218 INFO waiting for supervisor-proc-exit-listener, rsyslogd, teammgrd to die
Aug 25 14:53:17.235510 sonic NOTICE swss#portsyncd: :- onMsg: nlmsg type:16 key:PortChannel37 admin:0 oper:0 addr:1c:34:da:1d:42:00 ifindex:440 master:0 type:team
Aug 25 14:53:17.238131 sonic NOTICE swss#portsyncd: :- onMsg: nlmsg type:17 key:PortChannel37 admin:0 oper:0 addr:1c:34:da:1d:42:00 ifindex:440 master:0 type:team
Aug 25 14:53:18.221535 sonic INFO teamd#supervisord 2021-08-25 14:53:18,220 INFO reaped unknown pid 266 (exit status 0)
Aug 25 14:53:18.249512 sonic NOTICE swss#portsyncd: :- onMsg: nlmsg type:16 key:PortChannel38 admin:0 oper:0 addr:1c:34:da:1d:42:00 ifindex:441 master:0 type:team
Aug 25 14:53:18.251172 sonic NOTICE swss#portsyncd: :- onMsg: nlmsg type:17 key:PortChannel38 admin:0 oper:0 addr:1c:34:da:1d:42:00 ifindex:441 master:0 type:team
Aug 25 14:53:19.223408 sonic INFO teamd#supervisord 2021-08-25 14:53:19,222 INFO reaped unknown pid 274 (exit status 0)
Aug 25 14:53:19.262189 sonic NOTICE swss#portsyncd: :- onMsg: nlmsg type:16 key:PortChannel39 admin:0 oper:0 addr:1c:34:da:1d:42:00 ifindex:442 master:0 type:team
Aug 25 14:53:19.264974 sonic NOTICE swss#portsyncd: :- onMsg: nlmsg type:17 key:PortChannel39 admin:0 oper:0 addr:1c:34:da:1d:42:00 ifindex:442 master:0 type:team
Aug 25 14:53:20.226169 sonic INFO teamd#supervisord 2021-08-25 14:53:20,224 INFO reaped unknown pid 282 (exit status 0)
Aug 25 14:53:20.226169 sonic INFO teamd#supervisord 2021-08-25 14:53:20,224 INFO waiting for supervisor-proc-exit-listener, rsyslogd, teammgrd to die
Aug 25 14:53:20.275229 sonic NOTICE swss#portsyncd: :- onMsg: nlmsg type:16 key:PortChannel4 admin:0 oper:0 addr:1c:34:da:1d:42:00 ifindex:443 master:0 type:team
Aug 25 14:53:20.277616 sonic NOTICE swss#portsyncd: :- onMsg: nlmsg type:17 key:PortChannel4 admin:0 oper:0 addr:1c:34:da:1d:42:00 ifindex:443 master:0 type:team
Aug 25 14:53:21.228408 sonic INFO teamd#supervisord 2021-08-25 14:53:21,226 INFO reaped unknown pid 290 (exit status 0)
Aug 25 14:53:21.288364 sonic NOTICE swss#portsyncd: :- onMsg: nlmsg type:16 key:PortChannel40 admin:0 oper:0 addr:1c:34:da:1d:42:00 ifindex:444 master:0 type:team
Aug 25 14:53:21.291392 sonic NOTICE swss#portsyncd: :- onMsg: nlmsg type:17 key:PortChannel40 admin:0 oper:0 addr:1c:34:da:1d:42:00 ifindex:444 master:0 type:team
Aug 25 14:53:22.229471 sonic INFO teamd#supervisord 2021-08-25 14:53:22,228 INFO reaped unknown pid 298 (exit status 0)
Aug 25 14:53:22.302966 sonic NOTICE swss#portsyncd: :- onMsg: nlmsg type:16 key:PortChannel41 admin:0 oper:0 addr:1c:34:da:1d:42:00 ifindex:445 master:0 type:team
Aug 25 14:53:22.304387 sonic NOTICE swss#portsyncd: :- onMsg: nlmsg type:17 key:PortChannel41 admin:0 oper:0 addr:1c:34:da:1d:42:00 ifindex:445 master:0 type:team
Aug 25 14:53:23.232371 sonic INFO teamd#supervisord 2021-08-25 14:53:23,230 INFO reaped unknown pid 306 (exit status 0)
Aug 25 14:53:23.232371 sonic INFO teamd#supervisord 2021-08-25 14:53:23,231 INFO waiting for supervisor-proc-exit-listener, rsyslogd, teammgrd to die
Aug 25 14:53:23.314882 sonic NOTICE swss#portsyncd: :- onMsg: nlmsg type:16 key:PortChannel42 admin:0 oper:0 addr:1c:34:da:1d:42:00 ifindex:446 master:0 type:team
Aug 25 14:53:23.315961 sonic NOTICE swss#portsyncd: :- onMsg: nlmsg type:17 key:PortChannel42 admin:0 oper:0 addr:1c:34:da:1d:42:00 ifindex:446 master:0 type:team
Aug 25 14:53:24.234149 sonic INFO teamd#supervisord 2021-08-25 14:53:24,233 INFO reaped unknown pid 314 (exit status 0)
Aug 25 14:53:24.328259 sonic NOTICE swss#portsyncd: :- onMsg: nlmsg type:16 key:PortChannel43 admin:0 oper:0 addr:1c:34:da:1d:42:00 ifindex:447 master:0 type:team
Aug 25 14:53:24.330021 sonic NOTICE swss#portsyncd: :- onMsg: nlmsg type:17 key:PortChannel43 admin:0 oper:0 addr:1c:34:da:1d:42:00 ifindex:447 master:0 type:team
Aug 25 14:53:25.236370 sonic INFO teamd#supervisord 2021-08-25 14:53:25,235 INFO reaped unknown pid 322 (exit status 0)
Aug 25 14:53:25.336114 sonic NOTICE swss#portsyncd: :- onMsg: nlmsg type:16 key:PortChannel44 admin:0 oper:0 addr:1c:34:da:1d:42:00 ifindex:448 master:0 type:team
Aug 25 14:53:25.337225 sonic NOTICE swss#portsyncd: :- onMsg: nlmsg type:17 key:PortChannel44 admin:0 oper:0 addr:1c:34:da:1d:42:00 ifindex:448 master:0 type:team
Aug 25 14:53:26.239198 sonic INFO teamd#supervisord 2021-08-25 14:53:26,237 INFO reaped unknown pid 330 (exit status 0)
Aug 25 14:53:26.239198 sonic INFO teamd#supervisord 2021-08-25 14:53:26,238 INFO waiting for supervisor-proc-exit-listener, rsyslogd, teammgrd to die
Aug 25 14:53:26.350020 sonic NOTICE swss#portsyncd: :- onMsg: nlmsg type:16 key:PortChannel45 admin:0 oper:0 addr:1c:34:da:1d:42:00 ifindex:449 master:0 type:team
Aug 25 14:53:26.351829 sonic NOTICE swss#portsyncd: :- onMsg: nlmsg type:17 key:PortChannel45 admin:0 oper:0 addr:1c:34:da:1d:42:00 ifindex:449 master:0 type:team
Aug 25 14:53:27.240158 sonic INFO teamd#supervisord 2021-08-25 14:53:27,239 INFO reaped unknown pid 338 (exit status 0)
Aug 25 14:53:27.262915 sonic NOTICE swss#portsyncd: :- onMsg: nlmsg type:16 key:PortChannel46 admin:0 oper:0 addr:1c:34:da:1d:42:00 ifindex:450 master:0 type:team
Aug 25 14:53:27.265131 sonic NOTICE swss#portsyncd: :- onMsg: nlmsg type:17 key:PortChannel46 admin:0 oper:0 addr:1c:34:da:1d:42:00 ifindex:450 master:0 type:team
Aug 25 14:53:28.242412 sonic INFO teamd#supervisord 2021-08-25 14:53:28,241 INFO reaped unknown pid 346 (exit status 0)
Aug 25 14:53:28.274277 sonic NOTICE swss#portsyncd: :- onMsg: nlmsg type:16 key:PortChannel47 admin:0 oper:0 addr:1c:34:da:1d:42:00 ifindex:451 master:0 type:team
Aug 25 14:53:28.276417 sonic NOTICE swss#portsyncd: :- onMsg: nlmsg type:17 key:PortChannel47 admin:0 oper:0 addr:1c:34:da:1d:42:00 ifindex:451 master:0 type:team
Aug 25 14:53:29.245317 sonic INFO teamd#supervisord 2021-08-25 14:53:29,244 INFO reaped unknown pid 354 (exit status 0)
Aug 25 14:53:29.245317 sonic INFO teamd#supervisord 2021-08-25 14:53:29,244 INFO waiting for supervisor-proc-exit-listener, rsyslogd, teammgrd to die
Aug 25 14:53:29.287874 sonic NOTICE swss#portsyncd: :- onMsg: nlmsg type:16 key:PortChannel48 admin:0 oper:0 addr:1c:34:da:1d:42:00 ifindex:452 master:0 type:team
Aug 25 14:53:29.288887 sonic NOTICE swss#portsyncd: :- onMsg: nlmsg type:17 key:PortChannel48 admin:0 oper:0 addr:1c:34:da:1d:42:00 ifindex:452 master:0 type:team
Aug 25 14:53:30.247470 sonic INFO teamd#supervisord 2021-08-25 14:53:30,246 INFO reaped unknown pid 364 (exit status 0)
Aug 25 14:53:30.300217 sonic NOTICE swss#portsyncd: :- onMsg: nlmsg type:16 key:PortChannel49 admin:0 oper:0 addr:1c:34:da:1d:42:00 ifindex:453 master:0 type:team
Aug 25 14:53:30.302218 sonic NOTICE swss#portsyncd: :- onMsg: nlmsg type:17 key:PortChannel49 admin:0 oper:0 addr:1c:34:da:1d:42:00 ifindex:453 master:0 type:team
Aug 25 14:53:31.250303 sonic INFO teamd#supervisord 2021-08-25 14:53:31,248 INFO reaped unknown pid 372 (exit status 0)
Aug 25 14:53:31.311055 sonic NOTICE swss#portsyncd: :- onMsg: nlmsg type:16 key:PortChannel5 admin:0 oper:0 addr:1c:34:da:1d:42:00 ifindex:454 master:0 type:team
Aug 25 14:53:31.312501 sonic NOTICE swss#portsyncd: :- onMsg: nlmsg type:17 key:PortChannel5 admin:0 oper:0 addr:1c:34:da:1d:42:00 ifindex:454 master:0 type:team
Aug 25 14:53:32.252200 sonic INFO teamd#supervisord 2021-08-25 14:53:32,250 INFO reaped unknown pid 380 (exit status 0)
Aug 25 14:53:32.252200 sonic INFO teamd#supervisord 2021-08-25 14:53:32,251 INFO waiting for supervisor-proc-exit-listener, rsyslogd, teammgrd to die
Aug 25 14:53:32.324508 sonic NOTICE swss#portsyncd: :- onMsg: nlmsg type:16 key:PortChannel50 admin:0 oper:0 addr:1c:34:da:1d:42:00 ifindex:455 master:0 type:team
Aug 25 14:53:32.326228 sonic NOTICE swss#portsyncd: :- onMsg: nlmsg type:17 key:PortChannel50 admin:0 oper:0 addr:1c:34:da:1d:42:00 ifindex:455 master:0 type:team
Aug 25 14:53:33.253284 sonic INFO teamd#supervisord 2021-08-25 14:53:33,252 INFO reaped unknown pid 388 (exit status 0)
Aug 25 14:53:33.336905 sonic NOTICE swss#portsyncd: :- onMsg: nlmsg type:16 key:PortChannel51 admin:0 oper:0 addr:1c:34:da:1d:42:00 ifindex:456 master:0 type:team
Aug 25 14:53:33.338974 sonic NOTICE swss#portsyncd: :- onMsg: nlmsg type:17 key:PortChannel51 admin:0 oper:0 addr:1c:34:da:1d:42:00 ifindex:456 master:0 type:team
Aug 25 14:53:34.255238 sonic INFO teamd#supervisord 2021-08-25 14:53:34,254 INFO reaped unknown pid 396 (exit status 0)
Aug 25 14:53:34.347492 sonic NOTICE swss#portsyncd: :- onMsg: nlmsg type:16 key:PortChannel52 admin:0 oper:0 addr:1c:34:da:1d:42:00 ifindex:457 master:0 type:team
Aug 25 14:53:34.348497 sonic NOTICE swss#portsyncd: :- onMsg: nlmsg type:17 key:PortChannel52 admin:0 oper:0 addr:1c:34:da:1d:42:00 ifindex:457 master:0 type:team
Aug 25 14:53:35.257284 sonic INFO teamd#supervisord 2021-08-25 14:53:35,256 INFO reaped unknown pid 404 (exit status 0)
Aug 25 14:53:35.258110 sonic INFO teamd#supervisord 2021-08-25 14:53:35,257 INFO waiting for supervisor-proc-exit-listener, rsyslogd, teammgrd to die
Aug 25 14:53:35.359346 sonic NOTICE swss#portsyncd: :- onMsg: nlmsg type:16 key:PortChannel53 admin:0 oper:0 addr:1c:34:da:1d:42:00 ifindex:458 master:0 type:team
Aug 25 14:53:35.361360 sonic NOTICE swss#portsyncd: :- onMsg: nlmsg type:17 key:PortChannel53 admin:0 oper:0 addr:1c:34:da:1d:42:00 ifindex:458 master:0 type:team
Aug 25 14:53:36.260363 sonic INFO teamd#supervisord 2021-08-25 14:53:36,258 INFO reaped unknown pid 412 (exit status 0)
Aug 25 14:53:36.270686 sonic NOTICE swss#portsyncd: :- onMsg: nlmsg type:16 key:PortChannel54 admin:0 oper:0 addr:1c:34:da:1d:42:00 ifindex:459 master:0 type:team
Aug 25 14:53:36.272172 sonic NOTICE swss#portsyncd: :- onMsg: nlmsg type:17 key:PortChannel54 admin:0 oper:0 addr:1c:34:da:1d:42:00 ifindex:459 master:0 type:team
Aug 25 14:53:37.261138 sonic INFO teamd#supervisord 2021-08-25 14:53:37,260 INFO reaped unknown pid 420 (exit status 0)
Aug 25 14:53:37.284024 sonic NOTICE swss#portsyncd: :- onMsg: nlmsg type:16 key:PortChannel55 admin:0 oper:0 addr:1c:34:da:1d:42:00 ifindex:460 master:0 type:team
Aug 25 14:53:37.285142 sonic NOTICE swss#portsyncd: :- onMsg: nlmsg type:17 key:PortChannel55 admin:0 oper:0 addr:1c:34:da:1d:42:00 ifindex:460 master:0 type:team
Aug 25 14:53:38.263528 sonic INFO teamd#supervisord 2021-08-25 14:53:38,262 INFO reaped unknown pid 428 (exit status 0)
Aug 25 14:53:38.264351 sonic INFO teamd#supervisord 2021-08-25 14:53:38,263 INFO waiting for supervisor-proc-exit-listener, rsyslogd, teammgrd to die
Aug 25 14:53:38.296819 sonic NOTICE swss#portsyncd: :- onMsg: nlmsg type:16 key:PortChannel56 admin:0 oper:0 addr:1c:34:da:1d:42:00 ifindex:461 master:0 type:team
Aug 25 14:53:38.297902 sonic NOTICE swss#portsyncd: :- onMsg: nlmsg type:17 key:PortChannel56 admin:0 oper:0 addr:1c:34:da:1d:42:00 ifindex:461 master:0 type:team
Aug 25 14:53:39.265548 sonic INFO teamd#supervisord 2021-08-25 14:53:39,264 INFO reaped unknown pid 436 (exit status 0)
Aug 25 14:53:39.309881 sonic NOTICE swss#portsyncd: :- onMsg: nlmsg type:16 key:PortChannel57 admin:0 oper:0 addr:1c:34:da:1d:42:00 ifindex:462 master:0 type:team
Aug 25 14:53:39.311153 sonic NOTICE swss#portsyncd: :- onMsg: nlmsg type:17 key:PortChannel57 admin:0 oper:0 addr:1c:34:da:1d:42:00 ifindex:462 master:0 type:team
Aug 25 14:53:40.267566 sonic INFO teamd#supervisord 2021-08-25 14:53:40,267 INFO reaped unknown pid 444 (exit status 0)
Aug 25 14:53:40.320851 sonic NOTICE swss#portsyncd: :- onMsg: nlmsg type:16 key:PortChannel58 admin:0 oper:0 addr:1c:34:da:1d:42:00 ifindex:463 master:0 type:team
Aug 25 14:53:40.322176 sonic NOTICE swss#portsyncd: :- onMsg: nlmsg type:17 key:PortChannel58 admin:0 oper:0 addr:1c:34:da:1d:42:00 ifindex:463 master:0 type:team
Aug 25 14:53:41.270894 sonic INFO teamd#supervisord 2021-08-25 14:53:41,268 INFO reaped unknown pid 452 (exit status 0)
Aug 25 14:53:41.270894 sonic INFO teamd#supervisord 2021-08-25 14:53:41,269 INFO waiting for supervisor-proc-exit-listener, rsyslogd, teammgrd to die
Aug 25 14:53:41.332208 sonic NOTICE swss#portsyncd: :- onMsg: nlmsg type:16 key:PortChannel59 admin:0 oper:0 addr:1c:34:da:1d:42:00 ifindex:464 master:0 type:team
Aug 25 14:53:41.333746 sonic NOTICE swss#portsyncd: :- onMsg: nlmsg type:17 key:PortChannel59 admin:0 oper:0 addr:1c:34:da:1d:42:00 ifindex:464 master:0 type:team
Aug 25 14:53:42.272385 sonic INFO teamd#supervisord 2021-08-25 14:53:42,271 INFO reaped unknown pid 460 (exit status 0)
Aug 25 14:53:42.336812 sonic INFO dockerd[601]: time="2021-08-25T14:53:42.334519626Z" level=info msg="Container 484888a93a601dc6e9b7e6f26e2d3d8b8da750691db88333502f48138b10b032 failed to exit within 60 seconds of signal 15 - using the force"
Aug 25 14:53:42.342366 sonic NOTICE swss#portsyncd: :- onMsg: nlmsg type:16 key:PortChannel6 admin:0 oper:0 addr:1c:34:da:1d:42:00 ifindex:465 master:0 type:team
Aug 25 14:53:42.343189 sonic NOTICE swss#portsyncd: :- onMsg: nlmsg type:17 key:PortChannel6 admin:0 oper:0 addr:1c:34:da:1d:42:00 ifindex:465 master:0 type:team
Aug 25 14:53:42.484036 sonic INFO containerd[472]: time="2021-08-25T14:53:42.481970216Z" level=info msg="shim reaped" id=484888a93a601dc6e9b7e6f26e2d3d8b8da750691db88333502f48138b10b032
Aug 25 14:53:42.494143 sonic INFO dockerd[601]: time="2021-08-25T14:53:42.494020400Z" level=info msg="ignoring event" module=libcontainerd namespace=moby topic=/tasks/delete type="*events.TaskDelete"
Aug 25 14:53:42.502952 sonic INFO systemd[1]: var-lib-docker-containers-484888a93a601dc6e9b7e6f26e2d3d8b8da750691db88333502f48138b10b032-mounts-shm.mount: Succeeded.
Aug 25 14:53:42.538343 sonic INFO systemd[1]: var-lib-docker-overlay2-dc3caf5ca2ce7282cd5f1fd4579e34dca167cc245d910a434ac0d3f44af0f678-merged.mount: Succeeded.
Aug 25 14:53:42.569106 sonic INFO /container: docker cmd: wait for teamd
Aug 25 14:53:42.570871 sonic INFO /container: docker cmd: stop for teamd
Aug 25 14:53:42.571181 sonic DEBUG /container: container_stop: END
Aug 25 14:53:42.605801 sonic NOTICE admin: Stopped teamd service...
Aug 25 14:53:42.608982 sonic INFO systemd[1]: teamd.service: Succeeded.
Aug 25 14:53:42.610290 sonic INFO systemd[1]: Stopped TEAMD container.
Aug 25 14:53:42.613674 sonic INFO systemd[1]: Starting Update NTP configuration...
Aug 25 14:53:43.014306 sonic INFO ntp-config.sh[58014]: Disabling NTP long jump for reboot type cold ...
Aug 25 14:53:43.028943 sonic NOTICE ntpd[54828]: ntpd exiting on signal 15 (Terminated)
Aug 25 14:53:43.029131 sonic INFO systemd[1]: Stopping Network Time Service...
Aug 25 14:53:43.033471 sonic INFO systemd[1]: Finished Update NTP configuration.
Aug 25 14:53:43.042408 sonic INFO systemd[1]: ntp.service: Succeeded.
Aug 25 14:53:43.043050 sonic INFO systemd[1]: Stopped Network Time Service.
Aug 25 14:53:43.044961 sonic INFO systemd[1]: Starting Network Time Service...
Aug 25 14:53:43.601942 sonic INFO ntp-systemd-wrapper[58032]: /usr/lib/ntp/ntp-systemd-wrapper: 44: /usr/lib/ntp/ntp-systemd-wrapper: log_daemon_msg: not found
Aug 25 14:53:43.609199 sonic NOTICE ntpd[58032]: ntpd 4.2.8p12@1.3728-o (1): Starting
Aug 25 14:53:43.609467 sonic INFO ntpd[58032]: Command line: /usr/sbin/ntpd -p /var/run/ntpd.pid -x -u 106:112
Aug 25 14:53:43.611996 sonic INFO systemd[1]: Started Network Time Service.
Aug 25 14:53:43.613861 sonic INFO ntpd[58037]: proto: precision = 0.105 usec (-23)
Aug 25 14:53:43.614030 sonic WARNING ntpd[58037]: restrict 0.0.0.0: KOD does nothing without LIMITED.
Aug 25 14:53:43.614199 sonic WARNING ntpd[58037]: restrict ::: KOD does nothing without LIMITED.
Aug 25 14:53:43.614374 sonic INFO ntpd[58037]: Listen normally on 0 lo 127.0.0.1:123
Aug 25 14:53:43.614554 sonic INFO ntpd[58037]: Listen normally on 1 eth0 10.210.24.55:123
Aug 25 14:53:43.614742 sonic INFO ntpd[58037]: Listen normally on 2 lo [::1]:123
Aug 25 14:53:43.614938 sonic INFO ntpd[58037]: Listening on routing socket on fd #19 for interface updates
Aug 25 14:53:43.615107 sonic INFO ntpd[58037]: kernel reports TIME_ERROR: 0x41: Clock Unsynchronized
```